src/OneDrive.Sdk.Authentication.Xamarin.Android/ManagedProtection.cs

```diff
@@ -0,0 +1,310 @@
+//
```

Comment:
Is it possible to bring this in as a dependent package instead of copy/pasting the code?